client/components/inline-notice/style.scss

```diff
@@ -4,14 +4,28 @@
 	// increasing the specificity of the styles to override the Gutenberg ones
 	&#{&} {
 		margin: 0;
-		margin-bottom: $grid-unit-30;
+		margin-bottom: $grid-unit-20;
+		border-left: 0;
+		padding: 12px;
```

Comment:
These changes update our notice to be inline with the Woo design library inline notices. 

<p align="center">
<img width="396" alt="Screenshot 2024-07-26 at 1 40 13 PM" src="https://github.com/user-attachments/assets/ae75bec3-68d5-4940-9bd1-dfc3e2e2ab38">
</p> 

ref: Kofpo8jj4zDWXDsVdlSz0f-fi-4172_49449

There were only 2 places in the Stripe plugin where these inline notices were used. I reviewed both of them to confirm they look correct with these changes.



<p align="center">
<img width="400" alt="Screenshot 2024-07-26 at 1 41 56 PM" src="https://github.com/user-attachments/assets/d5c13a4a-85e7-4424-9cbf-a2fdbf12086d">
</br>
<sup>Notice when the Google and Apple pay buttons don't load in the <strong>Express checkouts</strong> customize settings.</sup>
</p> 

<p align="center">
<img width="400" alt="Screenshot 2024-07-26 at 1 44 30 PM" src="https://github.com/user-attachments/assets/9e8942b7-fb17-403e-b32a-2e790a828bad">
</br>
<sup>Notice under the payment method settings explaining customization options.</sup>
</p>